src/mlpack/methods/ann/callbacks/early_stop_at_min_loss.hpp

```diff
@@ -0,0 +1,126 @@
+/**
+ * @file early_stop_at_min_loss.hpp
+ * @author Marcus Edel
+ * @author Omar Shrit
+ *
+ * Implementation of the early stop at minimum loss callback function.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under
+ * the terms of the 3-clause BSD license.  You should have received a copy of
+ * the 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CALLBACKS_EARLY_STOP_AT_MIN_LOSS_HPP
+#define MLPACK_CALLBACKS_EARLY_STOP_AT_MIN_LOSS_HPP
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * Early stopping to terminate the optimization process early if the loss stops
+ * decreasing.
+ */
+class EarlyStopAtMinLoss
+{
+ public:
+  /**
+   * Set up the early stop at min loss class, which keeps track of the minimum
+   * loss and stops the optimization process if the loss stops decreasing.
+   *
+   * @param patienceIn The number of epochs to wait after the minimum loss has
```

Comment:
It looks like you need to document output too.

Reply:
Actually, this file is added by error, I have to remove it.